### PR TITLE
fix: whole slew of fixes for linter+

### PR DESCRIPTION
**Pull request by zmattingly**

### CODE REVIEW

File: client/src/Components/ReviewsChart.jsx [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+  } else if (percentage >= 60) {
+    return 'Mostly Positive';
+  } else if (percentage >= 50) {
+    return 'Mixed';
+  } else if (percentage >= 40) {
+    return 'Mostly Negative';
+  } else if (percentage >= 30) {
+    return 'Negative';
+  }
+  return 'Very Negative';
```
Comment (ericletsgo) [inline client/src/Components/ReviewsChart.jsx:32]: Consider using 'else'

File: client/src/components/ChartVisualization.jsx [hunk 2]
```diff
@@ -43,45 +46,55 @@ class ChartVisualization extends React.Component {
...
 
-        var rectX = Math.floor(rect.attr('x'));
-        var rectY = Math.floor(rect.attr('y'));
+        const rectX = Math.floor(rect.attr('x'));
+        const rectY = Math.floor(rect.attr('y'));
+
+        const formattedDate = new Date(datum.date).toLocaleDateString(
+          'en-US',
+          { year: 'numeric', month: 'long', day: 'numeric' }
+        )
```
Comment (ericletsgo) [inline client/src/components/ChartVisualization.jsx:81]: semi colon

Comment (ericletsgo) [reply client/src/components/ChartVisualization.jsx:81]: Ignore

File: client/src/components/ChartVisualization.jsx [hunk 3]
```diff
@@ -95,11 +108,19 @@ class ChartVisualization extends React.Component {
   render() {
     return (
       <div className="chart-visualization">
-        <svg ref={ elem => { this.svgElement = elem; } } />
+        <svg ref={ (elem) => { this.svgElement = elem; } } />
       </div>
     );
   }
-
 }
```
Comment (ericletsgo) [inline client/src/components/ChartVisualization.jsx:115]: Maybe semi

File: server/models/index.js [hunk 4]
```diff
@@ -1,31 +1,29 @@
...
-const path      = require('path');
+const fs = require('fs');
+const path = require('path');
 const Sequelize = require('sequelize');
-const basename  = path.basename(__filename);
-const env       = process.env.NODE_ENV || 'development';
-const config    = require(__dirname + '/../config/config.json')[env];
-const db        = {};
+const config = require('./../config/config.json')[process.env.NODE_ENV || 'development'];
+var pg = require('pg');
```
Comment (ericletsgo) [inline server/models/index.js]: let or const

**Review by ericletsgo — COMMENTED** [review]
Left some comments. Mostly just lint issues. If you feel they are good then go ahead and pull.

File: client/src/Components/ReviewsChart.jsx (hunk 1 above)
Comment (zmattingly) [reply client/src/Components/ReviewsChart.jsx:32]: Linter complained about my else at the end, said it was unnecessary.

File: client/src/components/ChartVisualization.jsx (hunk 3 above)
Comment (zmattingly) [reply client/src/components/ChartVisualization.jsx:115]: ? Class definitions aren't assignments so shouldn't use a semi-colon.

File: server/models/index.js (hunk 4 above)
Comment (zmattingly) [reply server/models/index.js]: Fixed.

**Review by ericletsgo — APPROVED** [review]

**Pull request merged**